docs/index.md

```diff
@@ -0,0 +1,52 @@
+# FAIRmat nyaml documentation
+
+The `nyaml` tool offers a user-friendly command-line interface for converting between `YAML` and `XML` representations of NeXus definitions written in the NeXus Definition Language (NXDL). While the official NeXus definitions are maintained in `XML,` authoring them in `YAML` provides a more concise and readable alternative. `nyaml` bridges this gap, allowing developers to work in `YAML` while ensuring compatibility with the official `XML` format.
+
+The tool supports both application definitions and base classes. It can be installed via `pip` and includes comprehensive documentation and tutorials to assist users in writing NeXus definitions using `YAML`.
```

Comment:
"The tool supports both application definitions and base classes." why to mention it when it supports all classes?